sdks/python/setup.py

```diff
@@ -219,11 +227,15 @@ def run(self):
     install_requires=REQUIRED_PACKAGES,
     python_requires=python_requires,
     test_suite='nose.collector',
-    tests_require=REQUIRED_TEST_PACKAGES,
+    tests_require=[
+        REQUIRED_TEST_PACKAGES,
+        INTERACTIVE_BEAM,
+    ],
     extras_require={
         'docs': ['Sphinx>=1.5.2,<2.0'],
         'test': REQUIRED_TEST_PACKAGES,
         'gcp': GCP_REQUIREMENTS,
+        'ib': INTERACTIVE_BEAM,
```

Comment:
how about `interactive` instead of `ib`. This leads to a more readable `pip install apache_beam[interactive]` instead of `pip install apache_beam[ib]`

Reply:
Agreed!